tests/generator/generator.cpp

```diff
@@ -1930,3 +1930,39 @@ TEST(Generator, modelWithComplexUnitsOutOfScope)
 
     EXPECT_EQ(fileContents("generator/cellml_slc_example/model.py"), generator->implementationCode());
 }
+
+TEST(Generator, segfaultWhenResolvingImports)
+{
```

Comment:
Not too worried, but is this a test that should be in the `Generator` ?

Reply:
Moved to modelflattening: [94394a8](https://github.com/cellml/libcellml/pull/1204/commits/94394a875d8790da1c4258087560cc6f4275bf85).